e2e/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "devDependencies": {
+    "@applitools/eyes-cypress": "3.12.0",
+    "@testing-library/cypress": "7.0.0",
+    "async": "3.2.0",
+    "aws-sdk": "2.751.0",
+    "axios": "0.20.0",
+    "chai": "4.2.0",
+    "chalk": "4.1.0",
+    "cross-env": "7.0.2",
+    "cypress": "5.1.0",
+    "cypress-file-upload": "4.1.1",
+    "cypress-multi-reporters": "1.4.0",
+    "cypress-plugin-tab": "1.0.5",
+    "cypress-wait-until": "1.7.1",
+    "deepmerge": "4.2.2",
+    "dotenv": "8.2.0",
+    "knex": "0.21.5",
+    "lodash.intersection": "4.4.0",
+    "lodash.mapkeys": "4.6.0",
+    "lodash.without": "4.4.0",
+    "lodash.xor": "4.5.0",
+    "mattermost-redux": "github:mattermost/mattermost-redux#3d1028034d7677adfda58e91b9a5dcaf1bc0ff99",
+    "mattermost-webapp": "github:mattermost/mattermost-webapp#40853eab863f321bd64d67efe1b986864b01278c",
+    "mime": "2.4.6",
+    "mime-types": "2.1.27",
+    "mocha": "8.1.3",
+    "mocha-junit-reporters": "1.23.6",
+    "mocha-multi-reporters": "1.1.7",
+    "mochawesome": "6.1.1",
+    "mochawesome-merge": "4.1.0",
+    "mochawesome-report-generator": "5.1.0",
+    "moment-timezone": "0.5.31",
+    "mysql": "2.18.1",
+    "path": "0.12.7",
+    "pg": "8.3.3",
+    "recursive-readdir": "2.2.2",
+    "shelljs": "0.8.4",
+    "uuid": "8.3.0",
+    "yargs": "16.0.3"
```

Comment:
Do we need /all/ of these just to run the cypress tests? Or can we rely on `cypress` to bring what it needs?

Reply:
@hanzei I added these dependencies back in, as they are needed to import `'mattermost-webapp/e2e/cypress/support.ts`. This is the same setup as the demo plugin. We could go the route of copying functions from the webapp repo, but I think this way is better as all of the utility functions are defined in one place.

I'm not sure what the best way to do this is, as we are adding a lot of "unnecessary" dependencies here.